wlib/RepositoryTreeNode.cpp

```diff
@@ -317,6 +317,17 @@ int CAttributeNode::GetDispImage()
 	case SYNTAX_CHECK_FAIL:
 		return IID_DOCUMENT_SYNTAXFAIL;
 	}
+
+	std::wstring fileType = m_attribute->GetType()->GetRepositoryCode();
+	if (boost::algorithm::iequals(fileType, ATTRIBUTE_TYPE_DUD))
+		return IID_DOCUMENT_DUD;
+	if (boost::algorithm::iequals(fileType, ATTRIBUTE_TYPE_ESDL))
+		return IID_DOCUMENT_ESDL;
+	else if (boost::algorithm::iequals(fileType, ATTRIBUTE_TYPE_KEL))
+		return IID_DOCUMENT_KEL;
+	else if (boost::algorithm::iequals(fileType, ATTRIBUTE_TYPE_SALT))
```

Comment:
Minor (just an FYI) - but there is probably no need for an ```iequal``` here, a plain ```equal``` would suffice since both are #defines (and switching them to "const char * const" would allow a compare by address...